connectors/services/job_execution.py

```diff
@@ -57,11 +58,22 @@ async def _sync(self, sync_job):
                 )
                 return
 
-        if connector.features.document_level_security_enabled():
-            if (
-                sync_job.job_type == JobType.ACCESS_CONTROL
-                and connector.last_access_control_sync_status == JobStatus.IN_PROGRESS
-            ):
+        if (
+            sync_job.job_type == JobType.ACCESS_CONTROL
+            and connector.features.document_level_security_enabled()
+        ):
+            (
+                is_platinum_license_enabled,
+                license_enabled,
+            ) = await self.connector_index.has_active_license_enabled(License.PLATINUM)
+
+            if not is_platinum_license_enabled:
+                logger.error(
+                    f"Minimum required Elasticsearch license: 'platinum'. Actual license: '{license_enabled.value}'.  Skipping access control sync execution..."
```

Comment:
For expired license the message will be:
```
Actual license: expired.
```

Reply:
Yes, that was intentional, as it doesn't require any explicit control flow, but I can also change that. Do you have a suggestion for a log message?

Reply:
I don't like to use `if...else...` for logging. Let's keep it. It's good enough for troubleshooting.